gamestonk_terminal/cryptocurrency/defi/terramoney_fcd_model.py

```diff
@@ -0,0 +1,273 @@
+GOV_STATUSES = ["voting", "deposit", "passed", "rejected"]
+VALIDATORS_COLUMNS = [
+    "validatorName",
+    "tokensAmount",
+    "votingPower",
+    "commissionRate",
+    "status",
+    "uptime",
+]
+
+
+def _make_request(endpoint: str) -> dict:
+    """Helper method handles terra fcd api requests. [Source: https://fcd.terra.dev/v1]
+
+    Parameters
+    ----------
+    endpoint: str
+        endpoint url
+    Returns
+    -------
+    dict:
+        dictionary with response data
+    """
+
+    url = f"https://fcd.terra.dev/v1/{endpoint}"
+    response = requests.get(
+        url, headers={"Accept": "application/json", "User-Agent": "GST"}
+    )
+    if not 200 <= response.status_code < 300:
+        raise Exception(f"fcd terra api exception: {response.text}")
+    try:
+        return response.json()
+    except Exception as e:
+        raise ValueError(f"Invalid Response: {response.text}") from e
+
+
+def _adjust_delegation_info(delegation: dict) -> dict:
+    """Helper method which removes redundant fields from delegation info dictionary,
+    and denominate value fields. [Source: https://fcd.terra.dev/v1]
+
+    Parameters
+    ----------
+    delegation:
+        dictionary object with delegation data e.g.
+
+    Returns
+    -------
+    dict
+        adjusted dictionary with delegation data
+    """
+
+    delegation_info = {}
+    for key, value in delegation.items():
+        if key in ["amountDelegated", "totalReward"]:
+            delegation_info[key] = denominate_number(value)
+        elif key in ["validatorAddress", "rewards"]:
+            continue
+        else:
+            delegation_info[key] = value
+    return delegation_info
+
+
+def get_staking_account_info(address: str = "") -> tuple[pd.DataFrame, str]:
+    """Get staking info for provided terra account [Source: https://fcd.terra.dev/swagger]
+
+    Parameters
+    ----------
+    address: str
+        terra blockchain address e.g. terra1jvwelvs7rdk6j3mqdztq5tya99w8lxk6l9hcqg
+    Returns
+    -------
+    tuple[pd.DataFrame, str]:
+        luna delegations and summary report for given address
+    """
+
+    response = _make_request(f"staking/{address}")
+    results: dict[str, Any] = {"myDelegations": []}
+
+    for field in ["availableLuna", "delegationTotal"]:
+        results[field] = denominate_number(response.get(field, 0))
+
+    my_delegations = response.get("myDelegations")
+    if my_delegations:
+        for delegation in my_delegations:
+            validator = _adjust_delegation_info(delegation)
+            results["myDelegations"].append(validator)
+
+    df = pd.DataFrame(results["myDelegations"])
+
+    try:
+        df["validatorName"] = df["validatorName"].apply(lambda x: replace_unicode(x))
+        df.columns = prettify_column_names(list(df.columns))
+    except KeyError:
+        df = pd.DataFrame()
+
+    results["totalRewards"] = denominate_number(
+        response.get("rewards", {}).get("total", 0)
+    )
+
+    report = "Overview: \n"
```

Comment:
any specific reason for this not to be a multiline formatted string?
```python
report = f"""Overview:
   Address: {address}
   Available Luna: {results['availableLuna']}
   Delegated Luna: {results['delegationTotal']}
   Total Rewards:  {results['totalRewards']}"""
```

Reply:
Aaa I forget about it , will change it :)